bin/middlewares/unsupported_browser.js

```diff
@@ -0,0 +1,13 @@
+/* globals module */
+
+function isUnsupported(userAgent = '') {
+  return userAgent.match('Trident'); // old IE detection
+}
+
+module.exports = function(req, res, next) {
+  if (isUnsupported(req.headers['user-agent'])) {
+    res.redirect(302, '/unsupported');
```

Comment:
I think that we should define a feature flag on this redirection, to be able to merge this PR independently from the moment when the end of support is decided.

Reply:
Yes I should have done that in the first place… And it will be useful to the ensure translations of the page are complete.